modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/routes/RunQueryRoute.java

```diff
@@ -196,9 +193,10 @@ private RasSortField formatSortField(RasSortField sortValue) {
         return sortField;
     }
 
-    private List<RasRunResult> getRunsByIds(List<String> runIds, boolean isMethodDetailsRequested) throws InternalServletException {
+    private List<RasRunResult> getRunsByIds(List<String> runIds) throws InternalServletException {
         
-		boolean isShort = isTestRunShort(isMethodDetailsRequested);
+		boolean isShort = isTestRunShort(rasDetailsQueryParams.getDetail());
```

Comment:
isShort and isTestRunShort could confuse people - perhaps this should be something like isMethodDetailsExcluded? Or something better?